__tests__/components/sections/challenges/Objectives.test.tsx

```diff
@@ -0,0 +1,45 @@
+import Objectives from "@/components/sections/challenges/Objectives";
+import "@testing-library/jest-dom";
+import { screen } from "@testing-library/react";
+import { renderWithRedux } from "../../../../__mocks__/renderWithRedux";
+import { challenge as mockChallenge, submission } from "../../../../__mocks__/challenge";
+import DateManager from "@/utilities/DateManager";
+
+describe("Objectives", () => {
+  const challenge = mockChallenge();
+  it("should render objectives", () => {
+    renderWithRedux(<Objectives testId="objectiveId" />);
+    expect(screen.getByTestId("objectiveId")).toBeInTheDocument();
+    expect(screen.getByText("communities.overview.challenge.objective.title")).toBeInTheDocument();
+  });
+
+  it("should render the objective list", () => {
+    renderWithRedux(<Objectives />, { challenges: { current: mockChallenge(), list: [mockChallenge()], loading: false, submission: submission() } });
```

Comment:
can you create a mock object for the data and use it everywhere it is needed instead of doing this